.travis.yml

```diff
@@ -38,7 +38,7 @@ matrix:
     - env: TOX_ENV=flake8
       install: pip install flake8
```

Comment:
You can probably remove this as well if you want to use the tox venv. I remember doing this to make it run faster than having to install all the dependencies.